src/cuckoo.c

```diff
@@ -374,6 +374,7 @@ size_t CuckooFilter_Compact(CuckooFilter *cf) {
     return ret;
 }
 
+/* CF.DEBUG uses another function
```

Comment:
@ashtul why do you keep dead code?

Reply:
It is an info function which doesn't hurt to keep for possible future use.